src/game/server/entities/growingexplosion.h

```diff
@@ -15,6 +15,7 @@ enum
 	GROWINGEXPLOSIONEFFECT_LOVE_INFECTED,
 	GROWINGEXPLOSIONEFFECT_BOOM_INFECTED,
 	GROWINGEXPLOSIONEFFECT_HEAL_HUMANS,
+	GROWINGEXPLOSIONEFFECT_MERC_INFECTED,
```

Comment:
why is this being changed here? How is it related to turrets and hero?

Reply:
otherwise merc bombs would no longer harm the owner of the bomb